index.html

```diff
@@ -1980,6 +1980,22 @@ <h2>Disputes</h2>
 
       </section>
 
+      <section>
+        <h2>Authorization</h2>
+
+        <p>
+It is arguable that <a>verifiable credentials</a> or
+<a>verifiable presentations</a> should be used as authorization mechanisms
+that a <a>holder</a> could use to access various
+systems. This specification does not contemplate such a usage of verifiable
```

Comment:
```suggestion
systems. This specification MUST NOT be considered an authorization framework on its own.
```

Reply:
How about

Verifiable credentials are intended as a means of reliably identifying subjects. Whilst it is recognized that role based access controls (RBAC) and attribute based access controls (ABAC) rely on this identification as a means of authorizing subjects to access resources, this ?recommendation|specification? should not be used for authorization purposes without an accompanying authorization framework. This is because complex authorization issues such as delegation of authority are not addressed in this recommendation.

Reply:
+1, I like this.

Reply:
@brentzundel only issue with your MUST NOT is that it's not testable and we try to not put RFC language on non-testable things. That said, my point is a nitpick and I'd like us to take a stronger position on it... so we could just put it in and remove the language if anyone complains.